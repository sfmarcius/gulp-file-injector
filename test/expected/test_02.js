function test_01() {
    return "test_01";
}
function test_02() {
    return "test_02";
}
