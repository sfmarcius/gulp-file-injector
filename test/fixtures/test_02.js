//! <file path="./test_01.js" />
function test_02() {
    return "test_02";
}
