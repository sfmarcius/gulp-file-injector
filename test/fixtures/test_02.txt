$file(./test_01.txt)
test_02() {
    return "test_02";
}
