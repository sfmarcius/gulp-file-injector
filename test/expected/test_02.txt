test_01() {
    return "test_01";
}
test_02() {
    return "test_02";
}
